python/utils/bootstrap_utilities.py

```diff
@@ -148,6 +148,11 @@ def start_engine(self):
                 os.environ["SHOTGUN_DESKTOP_CURRENT_USER"]
```

Comment:
This is actually for Line 81:
Does this need porting, I would have assumed it would have been covered in the Desktop python 3 support?
It's using `unicode`?

Reply:
I don't understand the question. What is the problem exactly?

Reply:
The serialize/unserialize user method is python2/3 compliant.

Reply:
ugh, sorry I didn't read it properly, I just saw hound complaining about unicode earlier and didn't spot the check right next to it for Python 2.